src/SDK/Client/Platform/broadcastStateTransition.ts

```diff
@@ -1,36 +1,34 @@
+import crypto from "crypto";
 import { Platform } from "./Platform";
-import { wait } from "../../../utils/wait";
+import { StateTransitionBroadcastError } from "../../../errors/StateTransitionBroadcastError";
 
 /**
  * @param {Platform} platform
  * @param stateTransition
- * @param identity
- * @param {number} [keyIndex=0]
  */
-export default async function broadcastStateTransition(platform: Platform, stateTransition: any, identity: any, keyIndex : number = 0) {
+export default async function broadcastStateTransition(platform: Platform, stateTransition: any) {
     const { client, dpp } = platform;
 
-    const account = await client.getWalletAccount();
-
-    // @ts-ignore
-    const { privateKey } = account.getIdentityHDKeyById(
-        identity.getId().toString(),
-        keyIndex,
-    );
-
-    stateTransition.sign(
-        identity.getPublicKeyById(keyIndex),
-        privateKey,
-    );
-
     const result = await dpp.stateTransition.validateStructure(stateTransition);
 
     if (!result.isValid()) {
         throw new Error(`StateTransition is invalid - ${JSON.stringify(result.getErrors())}`);
     }
 
+    // Subscribing to future result
+    const hash = crypto.createHash('sha256')
```

Comment:
What is the point of having `hash` method that we can't use? Shall we change it to just sha256? Or change to blake2a everywhere since we use it for trees?

Reply:
Let's keep it for now and investigate this problem later.